app/components/header.jsx

```diff
@@ -33,37 +30,29 @@ export class Header extends Component {
   }
 
   render () {
-    const { session, showShare, user } = this.props;
+    const { dex, profile, session, showShare, user } = this.props;
     const ownPage = session && session.id === user.id;
 
     return (
-      <header>
-        <h1>{ownPage ? null : 'Viewing '}{user.username}'s Living Dex</h1>
-        <div className="share-container">
-          <a onClick={(e) => this.toggleShare(e, !showShare)}><i className="fa fa-link" /></a>
-          <a href={`http://twitter.com/home/?status=Check out ${ownPage ? 'my' : `${user.username}'s`} living dex progress on @PokedexTracker! https://pokedextracker.com/u/${user.username}`} target="_blank" onClick={() => ReactGA.event({ action: 'click tweet', category: 'Share' })}><i className="fa fa-twitter" /></a>
-          <ShareComponent show={showShare} username={user.username} />
-        </div>
-
-        <h2>
-          FC: <span className={user.friend_code ? '' : 'fc-missing'}>{user.friend_code || 'XXXX-XXXX-XXXX'}</span>
-          {ownPage ? <Link to="/account" onClick={() => ReactGA.event({ action: 'click edit friend code', category: 'User' })}><i className="fa fa-pencil" /></Link> : null}
-        </h2>
-
-        <RegionComponent />
-
-        <div className="percentage">
-          <ProgressComponent />
-          <RegionComponent mobile={true} />
-        </div>
-      </header>
+      <div>
```

Comment:
unless it's needed for stylez, this `div` isnt necessary since the `h1` can act as the container